src/screencast_keys/ops.py

```diff
@@ -523,17 +537,14 @@ class SK_OT_ScreencastKeys(bpy.types.Operator):
     # TODO: Clear when this model is finished.
     area_spaces = collections.defaultdict(set)
 
-    # Check if this operator is running.
-    # TODO: We can check it with the valid of event handler.
-    running = False
+    # Should screencast be restarted
+    restart: bpy.props.BoolProperty(
+        default=False
+    )
```

Comment:
```suggestion
    # Should screencast be restarted
    restart = bpy.props.BoolProperty(
        default=False
    )
```

And, please add `@compat.make_annotations` under `@BlClassRegistry()` which enables the add-on compatible on Blender 2.7X.

Reply:
Changed.